kernel/src/engine/parquet_row_group_skipping/tests.rs

```diff
@@ -59,13 +59,16 @@ fn test_get_stat_values() {
     ]);
     let filter = RowGroupFilter::new(metadata.metadata().row_group(0), &columns);
 
-    assert_eq!(filter.get_rowcount_stat(), Some(5));
+    assert_eq!(filter.get_rowcount_stat(), Some(Scalar::from(5)));
```

Comment:
I think you need to make sure it's a long, so
```suggestion
    assert_eq!(filter.get_rowcount_stat(), Some(Scalar::from(5i64)));
```

Reply:
Turns out there were a few of these. Fixed them, and also changed them to use `.into()` like all the other code in this test.